app.py

```diff
@@ -264,5 +266,58 @@ def get_pref(topic):
     )
 
 
+@app.route(f"{URL_PREFIX}/onboarding", methods=["GET", "POST"])
+@auth.requires_login
+def onboarding_survey():
+    if auth.get_account_status() != "pending_onboarding_survey":
+        return redirect(url_for("home"))
+
+    if request.method == "POST":
+        print(request.form)
+
+        with DB_ENGINE.connect() as conn:
+            repo = DbDemographicsRepository(conn)
+            account_id = auth.get_account_id()
+            
+            demo = Demographics(
```

Comment:
We should probably do some validation here. I don't expect issues, but realistically if people are sending weird data with postman or html-editing we should handle it without crashing.

Reply:
Done